Formula/libetonyek.rb

```diff
@@ -37,6 +37,11 @@ def install
       system "make", "install"
     end
 
+    # The mdds pkg-config .pc file includes the API version in its name (ex. mdds-2.0.pc).
+    # We extract this from the filename programmatically and store it in mdds_api_version.
+    mdds_pc_file = (Formula["mdds"].share/"pkgconfig").glob("*.pc").to_s
+    mdds_api_version = File.basename(mdds_pc_file, File.extname(mdds_pc_file)).split("-")[1]
```

Comment:
```suggestion
    mdds_pc_file = (Formula["mdds"].share/"pkgconfig").glob("mdds-*.pc").first.to_s
    mdds_api_version = File.basename(mdds_pc_file, File.extname(mdds_pc_file)).split("-")[1]
```

`glob` returns an array so you should really take the first element of that.

Reply:
Fixed!